### PR TITLE
docs: convert all images to webp

### PR DESCRIPTION
**Pull request by VandeurenGlenn**
Squooshed all PNG images to webp (with the default settings except the effort put to 6)

The material gif has about 10% compression
The other gif about 10% compression and colors set to 10

### CODE REVIEW

File: docs/images/material-web.gif
Comment (asyncLiz) [inline docs/images/material-web.gif:1]: What changed in this gif?

File: docs/components/images/icon/hero.gif
Comment (asyncLiz) [inline docs/components/images/icon/hero.gif:1]: Same question here as to what changed in the .gif file

**Review by asyncLiz — CHANGES_REQUESTED** [review]
Can you post the settings you used to generate the webp files in the PR's description?

File: docs/images/material-web.gif
Comment (VandeurenGlenn) [reply docs/images/material-web.gif:1]: Just putted some compression on it.

File: docs/components/images/icon/hero.gif
Comment (VandeurenGlenn) [reply docs/components/images/icon/hero.gif:1]: This one has some compression also (around 10%) and 10 of the colors (if I remember correctly).

**Review by asyncLiz — APPROVED** [review]
LGTM, sorry for the delay

**Comment by VandeurenGlenn** [comment]
> LGTM, sorry for the delay

No worries, when ya busy ya busy ;)

**Review by asyncLiz — CHANGES_REQUESTED** [review]
It looks like `docs/components/images/progress/hero.webp` isn't playing anymore, could you take a look?

**Comment by VandeurenGlenn** [comment]
> It looks like `docs/components/images/progress/hero.webp` isn't playing anymore, could you take a look?

Oeps, maybe I missed that one and is a gif, I'll see what went wrong.

**Comment by VandeurenGlenn** [comment]
So it seems animated webp exists, I didn't know, converted that one back to original one, if that one needs to be smaller also it would be best to have the input files because the animations have different frame times.

**Review by asyncLiz — APPROVED** [review]

**Pull request merged**